manifests/install/pe_gem.pp

```diff
@@ -1,11 +1,13 @@
 # This class links the r10k binary for PE
-class r10k::install::pe_gem {
+class r10k::install::pe_gem (
+  $puppet_master,
+) {
 
   require git
 
   # Puppet Enterprise 3.8 ships code to manage this symlink on install
   # This conditional should not effect FOSS customers based on the fact
-  unless versioncmp($::pe_version, '3.8.0') >= 0 {
+  unless versioncmp($::pe_version, '3.8.0') >= 0 and $puppet_master {
```

Comment:
I do need clarification on this one though, This check is for only nodes that are 3.7 and below so they don't declare this symlink as currently the puppet_enterprise module in 3.8 and higher manages the creation of the symlink. Can you provide a little more detail of what happen without this.